languages/common-lisp/exercises/concept/lists/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+## 1. Making a new list
+
+- Lists can be created with `list`.
```

Comment:
Would it be possible to have the hints not spell out the solution (i.e. the function to use, but instead point to a resource describing the solution? See https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd